src/ripple/app/misc/NetworkOPs.cpp

```diff
@@ -64,8 +64,8 @@
 #include <ripple/resource/ResourceManager.h>
 #include <ripple/rpc/BookChanges.h>
 #include <ripple/rpc/DeliveredAmount.h>
+#include <ripple/rpc/ServerHandler.h>
 #include <ripple/rpc/impl/RPCHelpers.h>
```

Comment:
If we strictly follow the lexicographic order, shouldn't the import order be reversed?
alphabetically, `S` comes after `i`. While ordering the imports, are we keeping all the `impl` imports together?

Reply:
It depends on whether the lexicographic order is case insensitive or not.  Lower case "i" comes after capital "S".

Reply:
okay, didn't think of that :)